instagram-app/src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import { FaSearch, FaRegCompass, FaRegHeart, FaRegUser, FaInstagram} from 'react-icons/fa'
+import {IconContext} from 'react-icons'
+import PropTypes from 'prop-types'
+import './SearchBar.scss'
+
+
+export default class Searchbar extends Component {
+    render() {
+        return (
+            <nav
+                className="navbar navbar-expand-md navbar-light bg-white sticky-top justify-content-md-between"
+            >
+                <IconContext.Provider value={{size:'32px'}}>
+                    <a
+                        href="/index.html"
+                        className="navbar-brand"
+                    >
+                        <FaInstagram /> | Instagram
+                    </a>
+                    <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                        <span className="navbar-toggler-icon"></span>
+                    </button>
+                    <section className="collapse navbar-collapse d-md-flex justify-content-md-between" id="navbarSupportedContent">
+                        <section className="search">
+                            <FaSearch/>
+                            <input type="text" placeholder="Search" onChange={this.props.searchEdit} value={this.props.searchValue}/>
+                        </section>
+                        <section className="split">
+                            <section className="icons">
+                                <FaRegCompass/>
+                                <FaRegHeart/>
+                                <FaRegUser/>
+                                <button onClick={this.props.logout}

```

Comment:
Great job with the nesting keeping it easy for me to read, and understand exactly how the component is constructed before even looking at the rendering.